packages/toolpad-app/src/toolpad/AppEditor/PageEditor/EditorCanvasHost.tsx

```diff
@@ -155,8 +155,8 @@ export default React.forwardRef<EditorCanvasHostHandle, EditorCanvasHostProps>(
             return;
           }
 
-          const { code, metaKey, shiftKey } = event;
-          const undoShortcut = code === 'KeyZ' && metaKey;
+          const { code, metaKey, shiftKey, ctrlKey } = event;
+          const undoShortcut = code === 'KeyZ' && (metaKey || ctrlKey);
```

Comment:
Personally, I would recommend using destructuring at least often as possible as it makes it harder to run quick regex search on the codebase, e.g. to search for the other locations where we use `event.code`. 

```suggestion
          const undoShortcut = event.code === 'KeyZ' && (event.metaKey || event.ctrlKey);
```

---

I had a look at https://github.com/mui/mui-toolpad/pull/1225#issuecomment-1302728271. I should have seen this one coming (I forgot, what happens when you are no longer day & night in the codebase). We can't use `event.code`. Per https://github.com/mui/mui-x/pull/3660/files#r793084149, our current shortcuts only work with a QWERTY layout. I have an AZERTY layout. It's also broken with the Dvorak layout.

I would recommend:
```suggestion
          const { code, metaKey, shiftKey, ctrlKey } = event;
          const undoShortcut = event.key.toLowerCase() === 'z' && (metaKey || ctrlKey);
```

or

```suggestion
          const { code, metaKey, shiftKey, ctrlKey } = event;
          const undoShortcut = String.fromCharCode(event.keyCode) === 'Z' && (metaKey || ctrlKey);
```

Reply:
Oh, interesting find! Thanks, I will fix this